omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -12,10 +12,17 @@ events {
 }
 
 http {
+<%
+    time_format = node['private_chef']['nginx']['time_format']
+    if time_format != 'time_iso8601' and time_format != 'time_local'
+        time_format = 'time_iso8601'
+    end
+%>
```

Comment:
This can be a function in https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/libraries/nginx_erb.rb and can be called like https://github.com/chef/chef-server/blob/19f45ec1b4ce37494e221c23c0c7a209704cb808/omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb#L154